js/main.js

```diff
@@ -8,6 +10,28 @@ search.addEventListener('click', function (event) {
   xhr.open('GET', 'https://pokeapi.co/api/v2/pokemon/' + input);
   xhr.responseType = 'json';
   xhr.addEventListener('load', function () {
+    if (xhr.status === 200) {
+      $searchAddMember.classList.remove('hidden');
+      var $imageTitle = document.createElement('h2');
```

Comment:
I would also consider moving lines 15-25 into their own function. The reason is that when you want to construct a DOM tree again from local storage on page load you will need to loop through your pokemon and do DOM creation again for each one.

Reply:
eg
```

xhr.addEventListener('load', function () {
    if (xhr.status === 200) {
      $searchAddMember.classList.remove('hidden');
      pokemonTree(xhr.response):
}

function pokemonTree(pokemon) {
      $imageTitle.setAttribute('class', 'image-title');
      $imageTitle.textContent = input;
      $imageName.appendChild($imageTitle);
      var $addButton = document.createElement('i');
      $addButton.setAttribute('class', 'fa-solid fa-plus fa-2x add-image-icon');
      $imageName.appendChild($addButton);
      var $image = document.createElement('img');
      $image.setAttribute('class', 'search-img');
      $image.setAttribute('src', pokemon.sprites.front_default);
      $imageName.appendChild($image);
}
```